StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetGDPRConfirmFlowTests.swift

```diff
@@ -302,7 +302,6 @@ final class PaymentSheet_GDPR_ConfirmFlowTests: STPNetworkStubbingTestCase {
         let clientSecretResolved = expectation(description: "clientSecretResolved")
         var clientSecret: String!
         let intent = try await intentCreation(intentKind: intentKind,
-                                              elementsSession: elementsSession,
```

Comment:
Can you delete this arg from `func intentCreation`? 

Also nit the func should probably be named `createIntent`

Reply:
Yep. It can be deleted since it is no longer being used since elementsSessions is no longer attached to the `Intent`